packages/drivechain_client/lib/widgets/qt_page.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:sail_ui/sail_ui.dart';
+
+class QtPage extends StatelessWidget {
+  final Widget child;
+
+  const QtPage({super.key, required this.child});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color:context.isDarkMode
+          ? context.sailTheme.colors.background // Dark Qt-grey
+          : const Color.fromARGB(255, 240, 240, 240), // Light Qt-grey
```

Comment:
leaning towards just setting background here. to avoid things being custom? Open to changing the backgrorund color as well